backend/Recodo.Common/Dtos/User/UserRegisterDTO.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Recodo.Common.Dtos.User
+{
+    public class UserRegisterDTO:UserDTO
```

Comment:
```suggestion
    public class NewUserDto : UserDTO
```

Maybe something like this?